compiler/src/test/scala/org/scalajs/core/compiler/test/ScalaJSDefinedTest.scala

```diff
@@ -599,9 +599,9 @@ class ScalaJSDefinedTest extends DirectTest with TestHelpers {
     }
     """ hasErrors
     """
-      |newSource1.scala:7: error: A Scala.js-defined JS trait can only contain abstract members
+      |newSource1.scala:7: error: Members of Scala.js-defined JS traits must either be abstract, or their right-hand-side must be `js.undefined`.
```

Comment:
Ah good point! Yes, I think we should improve that message for methods with `()`.